src/app/client/projects/desktop/src/app/modules/offline/components/offline-help-videos/offline-help-videos.component.spec.ts

```diff
@@ -1,16 +1,53 @@
-import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { RouterTestingModule } from '@angular/router/testing';
+import { ActivatedRoute, Router } from '@angular/router';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { ResourceService, ConfigService, BrowserCacheTtlService } from '@sunbird/shared';
+import { TelemetryService, TelemetryModule } from '@sunbird/telemetry';
+import { async, ComponentFixture, TestBed, fakeAsync, tick } from '@angular/core/testing';
 
 import { OfflineHelpVideosComponent } from './offline-help-videos.component';
+import { CacheService } from 'ng2-cache-service';
 
-xdescribe('OfflineHelpVideosComponent', () => {
+describe('OfflineHelpVideosComponent', () => {
   let component: OfflineHelpVideosComponent;
   let fixture: ComponentFixture<OfflineHelpVideosComponent>;
-
+  let resourceServiceStub;
   beforeEach(async(() => {
+    const fakeActivatedRoute = {
+      snapshot: {
+        data: {
+          telemetry: {
+            env: 'help', pageid: 'help', type: 'view'
+          }
+        }
+      }
+    };
+    resourceServiceStub = {
+      instance: 'sunbird',
+      frmelmnts: {
+        instn: {
+          't0094': 'How do I add content to the {instance} desktop app when I am connected to the Internet?',
+          't0095': 'How do I add content to the {instance} desktop app when I am offline or using a pen drive?',
+          't0096': 'My Library: How and where can I find content in My Library?',
+          't0097': 'How do I copy content to my pen drive?'
+        }
+      }
+    };
+    const routerStub = {
+      navigateByUrl: jasmine.createSpy('navigateByUrl')
+    };
+
     TestBed.configureTestingModule({
-      declarations: [ OfflineHelpVideosComponent ]
+      declarations: [OfflineHelpVideosComponent],
+      imports: [TelemetryModule, HttpClientTestingModule, RouterTestingModule],
```

Comment:
add sharedModules in imports and remove TelemetryService, ConfigService, CacheService, BrowserCacheTtlService, from providers